File: examples/java/hand-flow/src/main/java/dev/angzarr/examples/handflow/HandPhase.java
package dev.angzarr.examples.handflow;

/**
 * Internal state machine phases for hand orchestration.
 */
public enum HandPhase {
    WAITING_FOR_START,
    DEALING,
    POSTING_BLINDS,
    BETTING,
    DEALING_COMMUNITY,
    DRAW,
    SHOWDOWN,
    AWARDING_POT,
    COMPLETE
}

File: examples/csharp/HandFlow/HandPhase.cs
namespace HandFlow;

/// <summary>
/// Internal state machine phases for hand orchestration.
/// </summary>
public enum HandPhase
{
    WaitingForStart,
    Dealing,
    PostingBlinds,
    Betting,
    DealingCommunity,
    Draw,
    Showdown,
    AwardingPot,
    Complete
}
